crypto/armcap.c

```diff
@@ -54,6 +55,32 @@ void _armv8_pmull_probe(void);
 # ifdef __aarch64__
 void _armv8_sha512_probe(void);
 unsigned int _armv8_cpuid_probe(void);
+static void _armv8_rng_probe(void);
+
+static size_t OPENSSL_rndr_asm(unsigned char *buf, size_t len);
+static size_t OPENSSL_rndrrs_asm(unsigned char *buf, size_t len);
+
+static size_t OPENSSL_rndr_wrapper(size_t (*func)(unsigned char *, size_t), unsigned char *buf, size_t len)
+{
+    size_t buffer_size;
+    int i;
+
+    for (i = 0; i < 8; i++) {
+	buffer_size = func(buf, len);
+	if (buffer_size == len)
+	    break;
+	usleep(5000);  //5000 microseconds (5 milliseconds)
+    }
+    return buffer_size;
+}
+
+size_t OPENSSL_rndr_bytes(unsigned char *buf, size_t len) {
```

Comment:
{ should be on the next line for these 2 functions

Reply:
fixed